samples/src/Microsoft.Azure.IIoT.App/src/wwwroot/css/site.css

```diff
@@ -270,6 +270,18 @@ a.adhocmargin {
     display: flex;
 }
 
+.activated-true {
+    font-weight: bold;
+}
+
+.seen-true {
```

Comment:
I would call it .enabled instead of .seen - it makes it clearer

Reply:
Renamed